packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -54,10 +58,40 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
   async bootstrap(): Promise<void> {
     this.#validateOptions();
     this.strapi = await this.options.getStrapi();
+    if (!this.strapi) {
+      throw new ProviderInitializationError('Could not access local strapi');
+    }
+
+    // uploading to a non-local file provider without included entities is useless (and potentially problematic) because we don't have the links, so don't allow it
+    if (
+      this.strapi.config.get('plugin.upload').provider !== 'local' &&
```

Comment:
we can also not upload to local IMO, it would be better to start disconnecting DTS from upload providers specific logic, and also it's more consistent